README.md

```diff
@@ -2,97 +2,123 @@
 
 See [Ubuntu's website](https://ubuntu.com/) for more information on Ubuntu.
 
+## Table of content
+
+<!-- TOC -->
+* [Overview](#overview)
+* [Before Starting](#before-starting)
+* [Getting Started](#getting-started)
+  * [After Running The Script](#after-running-the-script)
+* [Dual-Boot - Ubuntu 22.04 with Windows 11](#dual-boot---ubuntu-2204-with-windows-11)
+  * [Requirement](#requirement)
+  * [Preparing Windows for Ubuntu](#preparing-windows-for-ubuntu)
+    * [Shrink Windows partition](#shrink-windows-partition)
+    * [Windows Drive Encryption](#windows-drive-encryption)
+  * [Installing Ubuntu](#installing-ubuntu)
+* [Notes](#notes)
+* [Change log](#change-log)
+* [Notes](#notes-1)
+  * [Slack](#slack)
+<!-- TOC -->
+
 ## Overview
 
-This folder contains installation scripts for setting up an Ubuntu 22.04 Linux machine for developers at Belighted.
+This doc includes an installation script to set up Ubuntu 22.04 Linux for developers at Belighted.
 
 ## Before Starting
 
-* download the Ubuntu 22.04 ISO from [Ubuntu's website](https://ubuntu.com/download/desktop/thank-you?version=22.04.1&architecture=amd64);
-* flash the downloaded ISO to a USB stick. There are multiple ways of doing that. For example, you could use [balenaEtcher](https://www.balena.io/etcher/);
+**If your goal is to dual-boot Linux (Ubuntu 22.04) and Windows (11), refer to
+the [Dual-Boot - Ubuntu 22.04 with Windows 11](#dual-boot---ubuntu-2204-with-windows-11) section.**
+
+* download the Ubuntu 22.04 ISO
+  from [Ubuntu's website](https://ubuntu.com/download/desktop/thank-you?version=22.04.1&architecture=amd64);
+* flash the downloaded ISO to a USB stick. There are multiple ways of doing that. For example, you could
+  use [balenaEtcher](https://www.balena.io/etcher/);
 * boot your computer from the flashed USB stick and follow Ubuntu's installer;
 * once this is all done, reboot and log in to your new operating system;
-* that's it! You can now follow the instructions in the [Getting Started](#Getting-Started) section;
```

Comment:
see [this issue](https://gist.github.com/rachelhyman/b1f109155c9dafffe618) for guidelines on how to write markdown anchor tags. There should not be any uppercase characters in the link